bundles/org.dataflowanalysis.analysis/src/org/dataflowanalysis/analysis/core/FlowGraphCollection.java

```diff
@@ -47,6 +53,13 @@ public FlowGraphCollection(List<? extends AbstractTransposeFlowGraph> transposeF
         this.transposeFlowGraphs = transposeFlowGraphs;
         this.resourceProvider = resourceProvider;
     }
+    
+    /**
+     * 
+     */
+    public void setTransposeFlowGraphs(List<? extends AbstractTransposeFlowGraph> transposeFlowGraphs) {
+        this.transposeFlowGraphs = transposeFlowGraphs;
+    }
```

Comment:
Is this setter really needed?
If not, the transpose flow graphs can be immutable (so `final`)

Reply:
probably got auto generated, so i would remove it